ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-traffic-engineering.j2

```diff
@@ -0,0 +1,43 @@
+{% if router_traffic_engineering is arista.avd.defined %}
+
+## Router Traffic-Engineering
+{%     if router_traffic_engineering.segment_routing is arista.avd.defined %}
+
+### Segment Routing Summary
+
+- SRTE is enabled.
+{%         if router_traffic_engineering.segment_routing.colored_tunnel_rib is arista.avd.defined(true) %}
+
+- system-colored-tunnel-rib is enabled
+{%         endif %}
+{%         if router_traffic_engineering.segment_routing.policy_endpoints is arista.avd.defined %}
+
+#### SRTE Policies
+
+| Endpoint | Color | Preference | Name | Description | Label Stack | Index  | Weight | Explicit Null |
+| -------- | ----- | ---------- | ---- | ----------- | ----------- | ------ | ------ | ------------- |
+{%             for endpoint in router_traffic_engineering.segment_routing.policy_endpoints | arista.avd.natural_sort %}
+{%                 for color in endpoint.colors | arista.avd.natural_sort %}
+{%                     set name = color.name | arista.avd.default('-') %}
+{%                     set description = color.description | arista.avd.default('-') %}
+{%                     for pathgroup in color.path_group | arista.avd.natural_sort %}
+{%                         set pathgroup_preference = pathgroup.preference | arista.avd.default('-') %}
```

Comment:
Because we are setting the default value while printing for all values under this for loop and the next inner for loop.

Please use the below method:
(the indentation is not proper, just an example)
```
{%       if pathgroup.preference is arista.avd.defined %}
{%            set pathgroup_preference = pathgroup.preference %}
{%       endif %}
```